src/domain/api/client.ts

```diff
@@ -13,7 +13,7 @@ export default new ApolloClient({
       });
     } catch (e) {
       // User not authenticated
-      console.error(e);
+      // TODO: add error-handler
```

Comment:
Is this swallowing the error? I think we should leave the console.error otherwise we might spend hours scratching our heads